source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,130 @@ Below you can find a list of RW API resources that use the `application` field:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?sort=-name,+description
+```
+
+As a rule of thumb, you can sort RW API resources using the `sort` query parameter. Usually, sorting can be performed using any field from the resource schema, so be sure to check each resource's model reference to find which fields can be used for sorting. In some cases, you also have the possibility of sorting by fields that are not present in the resource model (e.g., when fetching datasets, you can sort by `user.name` and `user.role` to sort datasets by the name or role of the owner of the dataset) - check each resource's documentation to find out which additional sorting criteria you have available. Sorting by nested fields is not supported at the moment.
+
+Multiple sorting criteria can be used, separating them by commas. You can also specify the sorting order by prepending the criteria with either `-` for descending order or `+` for ascending order. By default, ascending order is assumed.
+
+Keep in mind that it’s up to each individual RW API service (dataset, widget, layer, etc) to define and implement the sorting mechanisms. Because of this, the examples above may not be true for all cases. Refer to the documentation of each resource and endpoint for more details on sorting.
+
+### Which services comply with these guidelines
+
+The following endpoints adhere to the Sorting conventions defined above:
+
+* [Get v2 areas endpoint](/index-rw.html#getting-all-user-areas)
+* [Get areas endpoint](/index-rw.html#get-user-areas)
+* [Get collections endpoint](/index-rw.html#getting-collections-for-the-request-user)
+* [Get dashboards endpoint](/index-rw.html#getting-all-dashboards)
+* [Get datasets endpoint](/index-rw.html#getting-all-datasets)
+* [Get layers endpoint](/index-rw.html#getting-all-layers)
+* [Get metadata endpoint](/index-rw.html#getting-all-metadata)
+* [Get widgets endpoint](/index-rw.html#getting-all-widgets)
+
+## Filtering
+
+> Example request filtering using a single condition:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?name=viirs
+```
+
+> Example request filtering using multiple conditions:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?name=birds&provider=cartodb
+```
+
+> Example request filtering by an array field using the `,` OR multi-value separator:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?application=rw,gfw
+```
+
+> Example request filtering by an array field using the `@` AND multi-value separator:
+
+```shell
+curl -X GET https://api.resourcewatch.org/v1/dataset?application=rw@gfw
+```
+
+Like in the case of sorting, most RW API resources allow filtering the returned results of list endpoints using query parameters. As a rule of thumb, you can use the API resource's fields as query parameter filters, as shown in the examples on the side. You can also combine different query parameters into a complex `and` logic filter.
```

Comment:
What about OR with comma for single-valued fields? Like https://api.resourcewatch.org/v1/dataset?name=viirs,modis to get datasets matching viirs or modis. Should explicitly call out support or lack thereof.

Reply:
That's implicitly supported, as we were discussing in the comment above, by using a regex like `?name=viirs|modis` - do you think we need to explicitly highlight this as allowed?

Reply:
I would just briefly call it out here. Something like: "Note that you can achieve a logical `or` by passing a regular expression with two disjoint options, like this: `?name=<substr_a>|<substr_b>`."